src/AutoLaunchMac.coffee

```diff
@@ -1,33 +1,56 @@
-applescript = require 'applescript'
+fs = require('fs')
+mkdirp = require('mkdirp')
+untildify = require('untildify')
 
-tellTo = 'tell application "System Events" to '
+dir = untildify('~/Library/LaunchAgents')
 
 module.exports =
+    getFile: (opts) ->
+        file = dir+opts.appName+'.plist'
```

Comment:
@noamokman I think there is a `/` missing here because I keep getting file paths like this: `/Users/sp/Library/LaunchAgentsMinecraft.plist`